src/template.html

```diff
@@ -23,20 +23,20 @@ <h3 class="recent-scores-title">Recent scores</h3>
         </div>
         <div class="leaderboard">
           <div class="score">Name: <span>100</span></div>
+          <!-- <div class="score">Name: <span>100</span></div>
           <div class="score">Name: <span>100</span></div>
           <div class="score">Name: <span>100</span></div>
           <div class="score">Name: <span>100</span></div>
           <div class="score">Name: <span>100</span></div>
-          <div class="score">Name: <span>100</span></div>
-          <div class="score">Name: <span>100</span></div>
+          <div class="score">Name: <span>100</span></div> -->
```

Comment:
- [ ] kindly always be sure that comments should not be in your pr but I think you need it in future remove or populating method 
- [ ]  kindly try to remove commented code as much as possible